Adapt to Obsidian 1.x

liu-qilong opened the pull request:
I have been using Yin and Yang for its minimalism and tasteful appearance for more than a year. However, Obsidian v1.0 with massive changes to the UI setting was released on October 13, 2022, which caused disorder in various old themes, including Yin and Yang. This fork aims at adapting this widely loved Obsidian theme to the current version. If you agree with the following adjustments, please consider accepting this pull request 🤝😄

#### Change List

Major changes are listed below:

- Note tabs related
	- Revise the note tab style to simple rounded rectangle.
	- Align the top-lines of the note tab and the sidebars.
- Element style related
	- Remove the underline of external links.
	- Adjust the font size in the theme selection drop-down manual.
	- Remove relationship lines in file explorer.
	- Remove relationship lines in outline pane.
	- Remove the highlight background of the active day in the calendar plugin.
	- Remove the annoying white rectangle in the embedded box.
- Borderlines related
	- Remove the right border of the left sidebar.
	- Remove the tab panel's bottom border.
	- Align the left-sidebar bottom in full-screen mode.
	- Search input related
		- Solve breaks of the focus prompt-input border.
		- Solve over-enlarged prompt-input.
		- Adjust prompt-input padding.
		- Aligned search icon in "search installed plugins".
		- Remove duplicate search input clear button.
- PDF export related
	- Revise external links' colour in pdf exporting.
	- Revise block quote colour and border in pdf exporting.
	- Revise heading 1~6 sizes in pdf exporting.
- Mobile app related
	- Adjust mobile workspace padding.
	- Adjust embedded box content padding.
	- Remove strange icon in view header bar.
	- Solve breaks of the focus prompt-input border.

#### Example Issues & Fixes

Issues:
<img width="1450" alt="i1" src="https://user-images.githubusercontent.com/77708899/195970851-da12e4ac-0691-461f-a152-cc311de0aead.png">

Fix:
<img width="1450" alt="f1" src="https://user-images.githubusercontent.com/77708899/195970856-10001314-ac55-4d5d-861b-d2eb64d9d5f0.png">

Issues:
<img width="1450" alt="i2" src="https://user-images.githubusercontent.com/77708899/195971077-43f22ac6-0797-4ec8-ae83-5f9f0d311d54.png">

Fix:
<img width="1450" alt="f2" src="https://user-images.githubusercontent.com/77708899/195977540-11943794-54e6-49c2-a9cf-080a8bf9a822.png">

liu-qilong committed:
adapt to obsidian v1.0

liu-qilong committed:
fix prompt-input

liu-qilong committed:
align search icon

liu-qilong committed:
restore shadow in prompt-input

liu-qilong committed:
fix embedded box issue

liu-qilong committed:
adjust mobile workspace padding

liu-qilong committed:
refine mobile elements

liu-qilong committed:
refine outline pane

liu-qilong committed:
remove duplicate search input clear button

liu-qilong committed:
remove underline when hover internal and external links

turing0 commented:
Nice, I like it!

liu-qilong committed:
enlarge line height in file explorer

Samsamsam21 commented:
Nice, thks!

protortyp commented:
Awesome, thank you so much! I'm using this theme every day 🥳

f3ss1 commented:
IDK really is it my problem or you forgot to fix it though :)

![image](https://user-images.githubusercontent.com/48713543/197416211-3694f810-6412-4c9a-b8d0-9dd0574a20ca.png)

That is how the close/minimize/restore down look for me in the upper right. Any ideas on this? :)

Had the same problem with the original one once 1.0 released, same with yours update. This theme only, others work ok-ish.

liu-qilong committed:
restore embed rounded box in v.1.1.8

liu-qilong committed:
fix embed box title alignment issue

liu-qilong committed:
adjust embed box max height to 600px

liu-qilong committed:
refine windows & linux title bottons padding

liu-qilong commented:
> IDK really is it my problem or you forgot to fix it though :)
> 
> ![image](https://user-images.githubusercontent.com/48713543/197416211-3694f810-6412-4c9a-b8d0-9dd0574a20ca.png)
> 
> That is how the close/minimize/restore down look for me in the upper right. Any ideas on this? :)
> 
> Had the same problem with the original one once 1.0 released, same with yours update. This theme only, others work ok-ish.

Thanks for your feedback. Recently I tested it on a Windows 11 machine and refined the title buttons as below:

- Refined the shape of the title buttons' backgrounds to rectangles.
- All backgrounds are transparent when the mouse doesn't hover over them.

![Pasted image 20221224115735](https://user-images.githubusercontent.com/77708899/209421054-06cb054f-319a-4872-8feb-374b99f47ef1.png)

- When hovering over the close button, the background is highlighted as red.

![Pasted image 20221224115831](https://user-images.githubusercontent.com/77708899/209421067-1890fd7e-6765-4b94-8bfb-057b5e14bc71.png)

Hoping that this can solve the issue.

liu-qilong committed:
add aim & usage to readme

425833137 commented:
There is still some problems. My obsidian version is 1.1.9
![image](https://user-images.githubusercontent.com/86283215/210523416-c58a66e5-f4c5-4af8-8412-1efb9b568ca4.png)
But it is much better than before😂👏🏻Thank you for  your working!

425833137 commented:
> There is still some problems. My obsidian version is 1.1.9 ![image](https://user-images.githubusercontent.com/86283215/210523416-c58a66e5-f4c5-4af8-8412-1efb9b568ca4.png) But it is much better than before😂👏🏻Thank you for your working!

and left sidebar as well
![image](https://user-images.githubusercontent.com/86283215/210524791-a31ecf1e-726c-4461-a765-379a55bfec66.png)

liu-qilong committed:
fix background disorder in light theme

liu-qilong commented:
> > There is still some problems. My obsidian version is 1.1.9 ![image](https://user-images.githubusercontent.com/86283215/210523416-c58a66e5-f4c5-4af8-8412-1efb9b568ca4.png) But it is much better than before😂👏🏻Thank you for your working!
> 
> and left sidebar as well ![image](https://user-images.githubusercontent.com/86283215/210524791-a31ecf1e-726c-4461-a765-379a55bfec66.png)

Thanks for your feedback! I usually use the dark mode so I didn't notice this issue before. It should have been fixed now. Please feel free to contact me if there is still something I neglect.

liu-qilong committed:
fix embedded box content overflow in page preview

liu-qilong committed:
fix popover image preview sizing issue

Also improve the solution for embedded box content overflow issue in page preview

liu-qilong committed:
remove duplicated checkbox checked symbol
